compiler/testData/codegen/box/reflection/callBy/inlineClassDefaultArguments.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND_FIR: JVM_IR
```

Comment:
FYI, this is not a regression. This test was added today w/o this opt-out.

Reply:
Sorry about that, I forgot to run the FIR tests before submitting the PR.

Reply:
No problem. Hope `:compiler:test` trigger FIR tests as well...

Reply:
@jsjeon AFAIK... `:compiler:test` will only run a subset of the JVM tests. You need to run `firCompilerTest` to run all the FIR-related tests, or `:compiler:fir:fir2ir:test` to run the box tests for FIR (along with other FIR2IR tests). `compilerTest` will run _all_ compiler-related tests, including FIR.

Reply:
Ah, good to know that, thanks.